### ISSUE TITLE
Autocomplete option

### ISSUE THREAD

**Issue by nitsuga111986** [djpate/jTag#8]
Hi, I would like to know if AUTOCOMPLETE could get the data from DB, and how...

Thanks in advance!

**Comment by djpate** [comment]
Yes it works with data from the DB but the db script has to be written in some server side language(php, ruby etc..)

**Comment by nitsuga111986** [comment]
Do you have some example? because I am trying to implement this part but it doesnt work, I did this:
$("#img").tag({
                defaultWidth:200,
                defaultHeight:200,
                autoShowDrag : false,
                showTag : 'hover',
                autoComplete:[
                    $.post("ajax.php",{'action':'tag_recover'},function(){
                        data;
                    })
                    ]
});

in the ajax file, I did this:

switch($_REQUEST['action']){

```
    case 'tag_recover':
        echo "'uncle1 jack','baby1 john'";
```

}

But when I write something in the input, I receive this error: (on firebug)

Permission denied http://localhost to charge the property XMLHttpRequest.channel
http://ajax.googleapis.com/ajax/libs/jquery/1.4.4/jquery.min.js
Line 28

**Comment by nitsuga111986** [comment]
Hi again DjPate, I write again, because I did not find the solution for solve my problem, do you know how could I get data from database to use in autocomplete?

Thanks in advance, 
best regards :D
